### PR TITLE
Add support for delays

### PR DESCRIPTION
**Pull request by addyosmani**
Changes:

* API now supports a delay (in seconds) with a test
* Code formatting

reviewer: @sindresorhus

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,25 +52,39 @@ $ a11y index.html
...
 
 ```sh
 $ a11y --viewport-size=800x600
 ```
 
-Verbose mode:
+### Set a custom delay before capturing the page
+
+Type: Number (seconds)
+Default: 0
```
Comment (sindresorhus) [inline README.md]: Maybe default to 1 second to fix most problems with it not being able to capture JS changes right after document load.

File: index.js [hunk 2]
```diff
@@ -27,7 +27,8 @@ module.exports = function (url, opts, cb) {
     opts = objectAssign({}, opts, {
         url: protocolify(url),
         width: viewportSize[0] || 1024,
-        height: viewportSize[1] || 768
+        height: viewportSize[1] || 768,
+        delay: opts.delay || 0
```
Comment (sindresorhus) [inline index.js]: `delay: 0`

Comment (addyosmani) [inline index.js]: Hmm. `delay: 0` (delay: 1 now) breaks my tests whilst `delay: opts.delay || 1` appears to work. Am I doing something silly? :)

Comment (sindresorhus) [inline index.js]: Oops, actually, `delay` should be in the first object:

```js
objectAssign({delay: 1}, opts ...
```

**Comment by addyosmani** [comment]
@sindresorhus PR updated per last comment :)

File: cli.js [hunk 3]
```diff
@@ -23,6 +23,7 @@ var cli = meow({
         'Options',
         '  --verbose                 Displays more information',
         '  --viewport-size=1024x768  Sets the viewport size'
```
Comment (sindresorhus) [inline cli.js]: missing comma

**Pull request closed without merging**